File: C/crescente.c
#include <stdio.h>

int main()
{
    int x, y;

    printf("Digite dois números:\n");
    scanf("%d,", &x);
    scanf("%d,", &y);

    while (x != y) {
        if (x < y) {
            printf("CRESCENTE\n");
        }
        else {
            printf("Decrescente\n");
        }

        printf("Digite outros dois números:\n");
        scanf("%d,", &x);
        scanf("%d,", &y);
    }
    return 0;
}

File: C#/crescente.cs
using System;

namespace crescente
{
	class Program
	{
		static void Main(string[] args)
		{
			int x, y;

			Console.WriteLine("Digite dois números:");
			x = int.Parse(Console.ReadLine());
			y = int.Parse(Console.ReadLine());

			while (x != y)
			{
				if (x > y)
				{
					Console.WriteLine("DECRESCENTE!");
				}
				else
				{
					Console.WriteLine("CRESCENTE!");
				}

				Console.WriteLine("Digite outros dois números:");
				x = int.Parse(Console.ReadLine());
				y = int.Parse(Console.ReadLine());
			}
		}
	}
}
